tests/README.md

```diff
@@ -21,3 +21,27 @@ Specific location could be provided to this command as an argument, either as ab
 (eg. `python ${OPENPYPE_ROOT}/start.py start.py runtests ../tests/integration`) will trigger only tests in `integration` folder.
 
 See `${OPENPYPE_ROOT}/cli.py:runtests` for other arguments.
+
+Run in IDE:
+-----------
+If you would prefer to run/debug single file dirrectly in IDE of your choice, you might encounter issues with imports.
+It would manifest like `KeyError: 'OPENPYPE_DATABASE_NAME'`. That means you are importing module that depends on OP to be running, eg. all expected variables are set.
+
+In some cases your tests might be so localized, that you don't care about all env vars to be set properly.
+In that case you might add this dummy configuration BEFORE any imports in your test file
```

Comment:
I like this being documented - but I'd love it even more if we'd be able to streamline running the tests even more so that we wouldn't even need to pass in the mockup data manually. Running tests should be made as easy as possible.

Reply:
I hope this will be sorted out after this refactoring and clean up that is in progress. I find this PITA too.